monolith/src/main/java/com/society/server/dto/post/CreatePostDTO.java

```diff
@@ -0,0 +1,32 @@
+package com.society.server.dto.post;
+
+import jakarta.validation.constraints.NotEmpty;
+
+public class CreatePostDTO {
+
+    @NotEmpty
+    private String textContent;
+
+    private String imageUrl;
+
+    public CreatePostDTO() {
+    }
+
+    public String getTextContent() {
+        return textContent;
+    }
+
+    public CreatePostDTO setTextContent(String textContent) {
```

Comment:
Why do we need return type to be `CreatePostDTO ` in the all setters it can be void?

Reply:
I think this type of setters are called Builders. They are used to bring a clear structure to the code.
Example: 
commentEntity.setCreatorUsername(username)
                .setCommentText(createCommentDTO.getCommentText())
                .setImageUrl(createCommentDTO.getImageUrl())
                    .setPostId(postEntity.getId());

Reply:
See lombok builder instead https://projectlombok.org/features/Builder